client/state/reader/conversations/follow-status.js

```diff
@@ -2,6 +2,8 @@
 /**
  * Follow status constants
  */
-export const CONVERSATION_FOLLOW_STATUS_FOLLOWING = 'F';
-export const CONVERSATION_FOLLOW_STATUS_NOT_FOLLOWING = null;
-export const CONVERSATION_FOLLOW_STATUS_MUTING = 'M';
+export const CONVERSATION_FOLLOW_STATUS = {
```

Comment:
i wonder if we should freeze this in dev mode?

Reply:
Good plan. Doesn't seem to be a performance hit in Chrome these days:

https://stackoverflow.com/a/23198265

I've added it 👍